Sources/StreamChatUI/Utils/ImageCDN.swift

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+public var cachingKeyProvider: CachingKeyProvider = StreamCachingKeyProvider()
```

Comment:
I'd love to see how far can we get if we used this inside `Compomnents`

Reply:
Working on it 👍🏻

Reply:
Tried multiple ways but I'm failing to pull it out of components without ExtraData on `UIImageView` the only way I can see right now is to subclass 
```
class StreamImageView<ExtraData: ExtraDataTypes>: UIImageVIew {}
```
and add `StreamImageView` to components ass well

Reply:
I will open another PR with implementing the `UIImageView` subclass so we can decide if we want it or not. this PR can come trough